src/HotChocolate/MongoDb/src/MongoDb/Filters/Convention/Extensions/FilterConventionDescriptorExtensions.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using HotChocolate.Data.Filters;
+
+namespace HotChocolate.MongoDb.Data.Filters
```

Comment:
namespace: HotChocolate.Data.MongoDb

Reply:
are we leaving this as HotChocolate.MongoDb.Data?

Reply:
Is Neo4J going to be similar?

Reply:
HotChocolate.Data.MongoDb

Reply:
HotChocolate.Data.Neo4J